db/seeds.rb

```diff
@@ -148,6 +148,8 @@
               config_value: { options: ['Pledge Limit Guidelines:', '1st trimester (7-12 weeks): $100', '2nd trimester (12-24 weeks): $300', 'Later care (25+ weeks): $600']}
 Config.create config_key: :language,
               config_value: { options: ['Spanish', 'French', 'Korean']}
+Config.create config_key: :language,
```

Comment:
Should this be exercising the new one instead?

Reply:
🤦‍♂️  oops